matrix_sdk_appservice/src/lib.rs

```diff
@@ -37,6 +43,7 @@
 //!     .unwrap();
 //!
 //! let appservice = Appservice::new(homeserver_url, server_name, registration).await.unwrap();
+//! // set event handler with `appservice.client().set_event_handler()` here
```

Comment:
Plan is to refactor that into a compiling `run_with_callback` example instead and commenting that using event handler is also available